project-folder/CNN.ipynb

```diff
@@ -2,9 +2,17 @@
  "cells": [
```

Comment:
<p>I think you should make it clearer in your markdown comments that this is the model which achieved 93% accuracy and the other model was only for testing (?) purposes</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/akaashsidhu/SM_project/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>